cluster/gce/config-default.sh

```diff
@@ -247,7 +247,14 @@ RUNTIME_CONFIG="${KUBE_RUNTIME_CONFIG:-}"
 
 if [[ "${KUBE_FEATURE_GATES:-}" == "AllAlpha=true" ]]; then
   RUNTIME_CONFIG="${KUBE_RUNTIME_CONFIG:-api/all=true}"
-  RUN_CONTROLLERS="${RUN_CONTROLLERS:-*,endpointslice}"
+fi
+
+# If feature gates includes AllAlpha or EndpointSlice
+if [[ ( "${KUBE_FEATURE_GATES:-}" == *"AllAlpha=true"* ) || ( "${KUBE_FEATURE_GATES:-}" == *"EndpointSlice=true"* ) ]]; then
+  # Verify that EndpointSlice feature gate has not been disabled
+  if [[ "${KUBE_FEATURE_GATES:-}" != *"EndpointSlice=false"* ]]; then
```

Comment:
why not merge into one conditional?
&& ( "${KUBE_F...} != ... )

Reply:
It just felt easier to understand when it was split out like that. The condition got really long with everything on a single line.

Reply:
@bowei Updated to use single line condition.